src/util-lua-ssh.c

```diff
@@ -136,3 +192,9 @@ int SCLuaLoadSshLib(lua_State *L)
     luaL_newlib(L, sshlib);
     return 1;
 }
+
+int SCLuaLoadHasshLib(lua_State *L)
+{
+    SCSshEnableHassh();
```

Comment:
What do you think of this ?

Reply:
This seems fine, as it matches the current behavior of enabling it for any keyword that requires it.

But that also means there is no way to disable hassh right?  Unrelated to this PR though.

Reply:
Cool, note that this is not the case in the current implementation

> But that also means there is no way to disable hassh right?

Looks like `RegisterSSHParsers` does not handle differently `app-layer.protocols.ssh.hassh` == auto or == false...